### DISCUSSION TITLE
Apple silicon: is only M1 supported ?

### DISCUSSION THREAD

**Discussion by boxerab** [pocl/pocl#1727] [General]
In the README.md the support matrix mentions Apple M1. Does this mean the entire M series, or only the M1 cpu ?

**Comment by pjaaskel** [comment]
@isuruf can perhaps answer that?

**Comment by isuruf** [comment]
Entire M series. M1 was the only cpu in the M series when the README was written.

**Reply by pjaaskel** [reply]
Can you update the README to reflect the current Mac status?

**Comment by boxerab** [comment]
Thanks.
